tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -5,6 +5,7 @@
 from cupy.cuda import driver
 from cupy.cuda import runtime
 import cupyx
+import cupyx.lapack
```

Comment:
Seems unrelated to this PR?

Reply:
Yeah for some reason local tests failed without adding this, I was puzzled too..

Reply:
I see... Then could you add a comment here? Contributors with a clever linter setup may remove it by chance 😅

Reply:
I am now confused why the linter wouldn't complain about unused imports... @emcastillo do you recall what error we ran into without this change?

Reply:
Sorry, I can't remember the exact reason. I got some weird import error when kicking the tests.
I will try to check if they run without this line in the most recent commit.

Reply:
@leofang tests pass now without this. It should be ok to remote it

Reply:
It seems flake8 overlooks this case. I guess `import cupyx.lapack` is considered in-use because `cupyx` is used in code.

Simple reproducer:

```
import os
import os.path
import sys  # flake8 only warns about this


print(os.uname())
```

Reply:
```suggestion
```